Sources/AWSCrypto/Digest.swift

```diff
@@ -0,0 +1,16 @@
+// Digest.swift
+// Replicating the CryptoKit framework interface for < macOS 10.15
+// written by AdamFowler 2020/01/30
+#if (os(macOS) || os(iOS) || os(watchOS) || os(tvOS))
```

Comment:
replace with `#if canImport(CommonCrypto)`?

Reply:
This doesn't exactly sync with the test in Package.swift `#if os(Linux)`.

Reply:
`#if !os(Linux)` instead. See comment below